.github/workflows/ci.yml

```diff
@@ -98,10 +98,40 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-cache-go-${{ env.go-version }}-
       - name: setup env
-        run: make
+        run: make install get-libs
```

Comment:
imo `get-libs` shouldn't be needed to be called called explicitly. If `golangci-lint-github-action` requires it, we could just add it there as a prerequisite (like for `make test` and other targets that need it).